README.md

```diff
@@ -62,6 +68,9 @@ Above Vim script is just a sample and isn't robust. I'm planning to create or co
     [vim-go](https://github.com/fatih/vim-go)) intergration for importing
     package or opening godoc in Vim.
 
+### Alternative
```

Comment:
Ummm... it's not related with this PR and it may helps but I wonder this paragraph is needed in the first place.

Can you create another PR for this change at least?

Reply:
I'll remove it for now